File: Algorithms/Java/268-missing_number.java
class Solution {
    public int missingNumber(int[] nums) {

       int val1 = 0;
       int val2 = 0;
       int n = nums.length;

       for(int i = 0; i <= n; i++){
        val1 = val1 + i;
       }

       for(int i: nums){
        val2 = val2 + i;
       }


       return val1 - val2;
        
    }
}

File: Algorithms/C++/268-missing_number.cpp
class Solution {
public:
    int missingNumber(vector<int>& nums) {
        
        int total = 0;
        int total2 = 0;

        for(int n: nums){
            total += n;
        }
        
        int n = nums.size();
        total2 = n * (n + 1)/2;

        return total2 - total;
        
    }
};